wagtail_extensions/templates/wagtail_extensions/partials/menu.html

```diff
@@ -0,0 +1,8 @@
+{% load wagtailcore_tags %}
+<ul class="navbar-nav nav-fill">
```

Comment:
I'd be inclined to let the classes be an argument to the template tag, to allow flexibility without having to override the whole template?

Reply:
I feel that wouldn't be a good 'separation of concerns'. I'd rather wrap those classes in a block tag and call it a day? Perhaps also remove the `nav-fill` from the default list... then you could just add that in stylesheet?

Reply:
OK.